admin/modules/banners/includes/class-template.php

```
@@ -517,14 +528,36 @@ private function get_cache_key() {
+	}
+
 	/**
 	 * Retrieve stored template.
 	 *
 	 * @return string
 	 */
 	public function get_stored() {
 		$stored = get_option( $this->get_cache_key(), array() );
-		return isset( $stored[ $this->language ] ) ? $stored[ $this->language ] : array();
+		if ( ! is_array( $stored ) ) {
+			return array();
```

PHPStan (2.1.54)

[error] Method FazCookie\Admin\Modules\Banners\Includes\Template::get_stored() should return string but returns array.


(return.type)